src/main/java/service/DataConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import core.basesyntax.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
+            String[] parts = line.split(",");
+            FruitTransaction.Operation operation
+                    = FruitTransaction.Operation.valueOf(parts.toString());
+            String fruit = Arrays.toString(parts);
+            int quantity = Integer.parseInt(Arrays.toString(parts));
```

Comment:
To parse the quantity, you should use `parts[2]` if the quantity is the third element in the CSV, and also ensure that it is a number before parsing. Currently, `Arrays.toString(parts)` will not give you a parsable integer.